MonsterAttack2/src/components/game-components/moves-panel.jsx

```diff
@@ -0,0 +1,17 @@
+import PropTypes from "prop-types";
+
+const MovesPanel = ({ moves }) => {
```

Comment:
nit - we can add export while defining constant as well
`export const abc = {}`